docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -140,20 +140,66 @@ Regardless of the way you submit your work, follow these steps.
 .. _ticket tracker: https://code.djangoproject.com/
 .. _Development dashboard: https://dashboard.djangoproject.com/
 
-Non-trivial contributions
-=========================
-
-A "non-trivial" contribution is one that is more than a small bug fix. It's a
-change that introduces new Django functionality and makes some sort of design
-decision.
-
-If you provide a non-trivial change, include evidence that alternatives have
-been discussed on the `Django Forum`_ or |django-developers| list.
-
-If you're not sure whether your contribution should be considered non-trivial,
-ask on the ticket for opinions.
-
-.. _Django Forum: https://forum.djangoproject.com/
+Contributions which require community feedback
+==============================================
+
+A patch that introduces Django functionality and makes some sort of design
+decision requires a wider community discussion. Especially if the approach
+involves a deprecation or potentially introducing breaking changes.
```

Comment:
Adding settings in the list of non-trivial patches as per @sarahboyce's suggestion, maybe something like
```
A patch that introduces Django functionality and makes a design
decision requires wider community discussion. Examples of this include
adding new settings, adding new features, deprecations, and potentially
breaking changes.
```
wrapped at 79 characters